mw-webapp/src/utils/GoalMetricsVisibilityWorker.ts

```diff
@@ -0,0 +1,44 @@
+import {localStorageWorker} from "src/utils/LocalStorage";
+
+/**
+ * Available goal metrics
+ */
+export enum GoalMetricsVisibility {
+  TRUE = "true",
+  FALSE = "false"
+}
```

Comment:
Enum as flag already exist in typescript. It called 'Boolean'

Reply:
Deleted